[rospy.client][INFO] 2016-06-08 21:56:32,271: init_node, name[/spawn_gazebo_model], pid[7484]
[xmlrpc][INFO] 2016-06-08 21:56:32,272: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2016-06-08 21:56:32,272: Started XML-RPC server [http://cmeintel-desktop:32923/]
[rospy.impl.masterslave][INFO] 2016-06-08 21:56:32,272: _ready: http://cmeintel-desktop:32923/
[rospy.registration][INFO] 2016-06-08 21:56:32,273: Registering with master node http://localhost:11311
[xmlrpc][INFO] 2016-06-08 21:56:32,273: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2016-06-08 21:56:32,275: ROS Slave URI: [http://cmeintel-desktop:32923/]
[rospy.init][INFO] 2016-06-08 21:56:32,275: registered with master
[rospy.rosout][INFO] 2016-06-08 21:56:32,275: initializing /rosout core topic
[rospy.rosout][INFO] 2016-06-08 21:56:32,290: connected to core topic /rosout
[rospy.internal][INFO] 2016-06-08 21:56:32,461: topic[/rosout] adding connection to [/rosout], count 0
[rospy.simtime][INFO] 2016-06-08 21:56:35,313: initializing /clock core topic
[rospy.simtime][INFO] 2016-06-08 21:56:35,319: connected to core topic /clock
[rosout][INFO] 2016-06-08 21:56:35,321: Loading model xml from ros parameter
[rosout][INFO] 2016-06-08 21:56:35,331: Waiting for service /gazebo/spawn_urdf_model
[rospy.internal][INFO] 2016-06-08 21:56:35,337: topic[/clock] adding connection to [http://cmeintel-desktop:45183/], count 0
[rosout][INFO] 2016-06-08 21:56:35,345: Calling service /gazebo/spawn_urdf_model
[rosout][INFO] 2016-06-08 21:56:40,118: Spawn status: SpawnModel: Successfully spawned model
[rospy.core][INFO] 2016-06-08 21:56:40,119: signal_shutdown [atexit]
[rospy.internal][INFO] 2016-06-08 21:56:40,121: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2016-06-08 21:56:40,121: topic[/clock] removing connection to http://cmeintel-desktop:45183/
[rospy.impl.masterslave][INFO] 2016-06-08 21:56:40,121: atexit
[rospy.internal][WARNING] 2016-06-08 21:56:40,121: Unknown error initiating TCP/IP socket to cmeintel-desktop:48187 (http://cmeintel-desktop:45183/): Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 619, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

